demos/browser/app/meetingV2/meetingV2.html

```diff
@@ -62,6 +62,10 @@ <h1 class="h3 mb-3 font-weight-normal">Join a meeting</h1>
               <input type="checkbox" id="webaudio" class="custom-control-input">
               <label for="webaudio" class="custom-control-label">Use WebAudio</label>
             </div>
+            <div class="custom-control custom-checkbox" style="text-align: left;">
```

Comment:
nit: Can we make this a category with ERROR, WARN, INFO, and DEBUG? :D

Reply:
Yeah makes sense

Reply:
Nice screenshot 👍